tests/e2e/logging/test_openshift-logging.py

```diff
@@ -0,0 +1,148 @@
+    sc_obj = helpers.create_storage_class(
+        interface_type=constants.CEPHBLOCKPOOL,
+        interface_name=cbp_obj.name,
+        secret_name=constants.DEFAULT_SECRET,
+        reclaim_policy="Delete"
+    )
+    assert sc_obj, f"Failed to create storage class"
+    assert obj.create_instance_in_clusterlogging(sc_name=sc_obj.name)
+
+    # Check the health of the cluster-logging
+    assert obj.check_health_of_clusterlogging()
+
+
+def teardown(sc_obj, cbp_obj):
+    """
+    The teardown will uninstall the openshift-logging from the cluster
+    """
+    sc_obj.delete()
+    cbp_obj.delete()
+    uninstall_cluster_logging()
+
+
+@pytest.mark.usefixtures(
+    test_fixture.__name__
+)
+class TestLogging_in_EFK_stack(E2ETest):
+    """
+    The class contains the testcases related to openshift-logging
+    """
+    @pytest.fixture()
+    def create_pvc_and_deploymentconfig_pod(self, request, pvc_factory):
+        """
+
+        """
+        def finalizer():
+            helpers.delete_deploymentconfig(pod_obj)
+
+        request.addfinalizer(finalizer)
+
+        # Create pvc
+        pvc_obj = pvc_factory()
+
+        # Create service_account to get privilege for deployment pods
+        sa_name = helpers.create_serviceaccount(pvc_obj.project.namespace)
+
+        helpers.add_scc_policy(sa_name=sa_name.name, namespace=pvc_obj.project.namespace)
+
+        pod_obj = helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_obj.name,
+            namespace=pvc_obj.project.namespace,
+            sa_name=sa_name.name,
+            dc_deployment=True
+        )
+        helpers.wait_for_resource_state(resource=pod_obj, state=constants.STATUS_RUNNING)
+        return pod_obj, pvc_obj
+
+    @pytest.mark.polarion_id("OCS-657")
+    @tier1
+    @retry(ModuleNotFoundError, 6, 300, 3)
+    def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentconfig_pod):
+        """
+        This function creates new project to verify logging in EFK stack
+        1. Creates new project
+        2. Creates PVC
+        3. Creates Deployment pod in the new_project and run-io on the app pod
+        4. Logs into the EFK stack and check for new_project
+        5. And checks for the file_count in the new_project in EFK stack
+        """
+
+        pod_obj, pvc_obj = create_pvc_and_deploymentconfig_pod
+
+        # Running IO on the app_pod
+        pod_obj.run_io(storage_type='fs', size=8000)
+
+        # Searching for new_project in EFK stack
+        pod_list = get_all_pods(namespace='openshift-logging')
+        elasticsearch_pod = [
+            pod.name for pod in pod_list if pod.name.startswith('elasticsearch')
+        ]
+        elasticsearch_pod_obj = get_pod_obj(
+            name=elasticsearch_pod[1], namespace='openshift-logging'
+        )
+        projects = elasticsearch_pod_obj.exec_cmd_on_pod(command='indices | grep project', out_yaml_format=True)
+        logger.info(projects)
```

Comment:
Looks like some info is missing from the test logs. logger.info(projects) returns the following log. 
```20:45:55 - MainThread - test_openshift-logging - INFO - Wed Sep  4 15:15:55 UTC 2019 health status index```

Reply:
The logs lines are longer, Please feel free to scroll it to the right corner. I copied these lines from the same logs that i had submitted earlier

```
20:45:55 - MainThread - test_openshift-logging - INFO - Wed Sep  4 15:15:55 UTC 2019 health status index                                                                                                   uuid                   pri rep docs.count docs.deleted store.size pri.store.size green  open   .kibana                                                                                                 -wL9LrqyT6K-CQUJH7R9bA   1   1          5            1          0              0 green  open   project.namespace-test-9f2cea7724f141b89b5a0555e6e81e83.2ce42c41-cf26-11e9-9770-06ea8ca295b4.2019.09.04 54DvlYfqSNCCyZoksRSoLA   3   1          2            0          0              0 green  open   .searchguard                                                                                            uC9Q7Sc-SnWhQGRQWJUczw   1   1          5            0          0              0 green  open   .kibana.647a750f1787408bf50088234ec0edd5a6a9b2ac                                                        f_gukGJ1T1Kr5lRL8h6c4g   1   1          3            0          0              0 green  open   project.namespace-test-27e636c2757c4e6a82504c395d6889aa.fcbe999e-cedd-11e9-9770-06ea8ca295b4.2019.09.04 edjdiu0kRtivvHIbNZiCHw   3   1          2            0          0              0 green  open   project.namespace-test-a7396618a00d46e69c10cb7e8af2d6e7.20e39ead-cf22-11e9-9770-06ea8ca295b4.2019.09.04 yomP3b8EQMSLCxjNLPjMvA   3   1          2            0          0              0 green  open   .operations.2019.09.04                                                                                  cLoXo8ReSJ21SuGpVG9cwg   3   1    3582955            0       8464           4185 green  open   project.namespace-test-879767e3ed9a424fb43776a4f5fd1e8c.1ffc09fe-cf19-11e9-9770-06ea8ca295b4.2019.09.04 -sQyXDGeTaW1yWXpg-1MAg   3   1          2            0          0              0```

Reply:
20:45:55 - MainThread - test_openshift-logging - INFO - Wed Sep  4 15:15:55 UTC 2019 health status index                                                                                                   uuid                   pri rep docs.count docs.deleted store.size pri.store.size green  open   .kibana                                                                                                 -wL9LrqyT6K-CQUJH7R9bA   1   1          5            1          0              0 green  open   project.namespace-test-9f2cea7724f141b89b5a0555e6e81e83.2ce42c41-cf26-11e9-9770-06ea8ca295b4.2019.09.04 54DvlYfqSNCCyZoksRSoLA   3   1          2            0          0              0 green  open   .searchguard                                                                                            uC9Q7Sc-SnWhQGRQWJUczw   1   1          5            0          0              0 green  open   .kibana.647a750f1787408bf50088234ec0edd5a6a9b2ac                                                        f_gukGJ1T1Kr5lRL8h6c4g   1   1          3            0          0              0 green  open   project.namespace-test-27e636c2757c4e6a82504c395d6889aa.fcbe999e-cedd-11e9-9770-06ea8ca295b4.2019.09.04 edjdiu0kRtivvHIbNZiCHw   3   1          2            0          0              0 green  open   project.namespace-test-a7396618a00d46e69c10cb7e8af2d6e7.20e39ead-cf22-11e9-9770-06ea8ca295b4.2019.09.04 yomP3b8EQMSLCxjNLPjMvA   3   1          2            0          0              0 green  open   .operations.2019.09.04                                                                                  cLoXo8ReSJ21SuGpVG9cwg   3   1    3582955            0       8464           4185 green  open   project.namespace-test-879767e3ed9a424fb43776a4f5fd1e8c.1ffc09fe-cf19-11e9-9770-06ea8ca295b4.2019.09.04 -sQyXDGeTaW1yWXpg-1MAg   3   1          2            0          0              0

Reply:
ah okay! :)